src/Commands/core/SiteInstallCommands.php

```diff
@@ -276,7 +276,7 @@ public function pre(CommandData $commandData)
             // a fallback name when '--sites-subdir' is not specified, but
             // only if the uri and the folder name match, and only if
             // the sites directory has already been created.
-            $dir = $this->getSitesSubdirFromUri($root, $aliasRecord->get('uri'));
+            $dir = StringUtils::lookupSiteDirFromUri($aliasRecord->get('uri'), $root);
```

Comment:
Could we fall back to 'default' if the URI does not have a matching entry in sites.php?

I have a single site in `sites/default` and I have `drush.yml` file that sets 'options.uri: http://example.com`, I expect `drush si` to install the site even if I don't have a `sites.php` file.